packages/react-next/src/components/Checkbox/useCheckboxClasses.tsx

```diff
@@ -0,0 +1,26 @@
+import * as classes from './Checkbox.scss';
+import { createClassResolver, ClassFunction } from '@fluentui/react-compose';
+import { useGlobalClassNames } from '../../Styling/useGlobalClassNames';
+import { css } from '../../Utilities';
+import { ICheckboxClasses } from './Checkbox.types';
+
+const GlobalClassNames: Omit<ICheckboxClasses, 'input'> = {
+  root: 'ms-Checkbox',
+  container: 'ms-Checkbox-label',
+  checkbox: 'ms-Checkbox-checkbox',
+  checkmark: 'ms-Checkbox-checkmark',
+  text: 'ms-Checkbox-text',
+};
+
+const defaultClassResolver = createClassResolver(classes);
```

Comment:
If we *just* use `createClassResolver` without all this extra global classnames overhead, will that improve perf? (remove all the papercuts.)